thorlcr/graph/thgraph.cpp

```diff
@@ -1727,6 +1727,13 @@ void CGraphBase::abort(IException *e)
 
         abortException.set(e);
         aborted = true;
+
+        if (!job.getOptBool("coreOnAbort", false))
```

Comment:
Does this seem reasonable to get the option ?

Reply:
looks fine. Don't actually need the optional default value of false, since false is the default anyway.

Reply:
left as is to know the default easily.